[0.253s] Invoking command in '/home/jeanho/gps_rtk/build/ublox_gps': AMENT_PREFIX_PATH=/home/jeanho/gps_rtk/install/ublox_msgs:/home/jeanho/gps_rtk/install/ublox_serialization:/home/jeanho/gps_rtk/install/rtcm_msgs:${AMENT_PREFIX_PATH} CMAKE_PREFIX_PATH=/home/jeanho/gps_rtk/install/ublox_msgs:/home/jeanho/gps_rtk/install/ublox_serialization:/home/jeanho/gps_rtk/install/rtcm_msgs:/opt/ros/humble LD_LIBRARY_PATH=/home/jeanho/gps_rtk/install/ublox_msgs/lib:/home/jeanho/gps_rtk/install/rtcm_msgs/lib:/opt/ros/humble/opt/rviz_ogre_vendor/lib:/opt/ros/humble/lib/x86_64-linux-gnu:/opt/ros/humble/lib:/home/jeanho/acados/lib PYTHONPATH=/home/jeanho/gps_rtk/install/ublox_msgs/local/lib/python3.10/dist-packages:/home/jeanho/gps_rtk/install/rtcm_msgs/local/lib/python3.10/dist-packages:${PYTHONPATH} /usr/bin/cmake --build /home/jeanho/gps_rtk/build/ublox_gps -- -j24 -l24
[0.269s] -- Found ament_cmake_ros: 0.10.0 (/opt/ros/humble/share/ament_cmake_ros/cmake)
[0.344s] -- Found diagnostic_msgs: 4.9.0 (/opt/ros/humble/share/diagnostic_msgs/cmake)
[0.468s] -- Found rosidl_generator_c: 3.1.7 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.469s] -- Found rosidl_adapter: 3.1.7 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.471s] -- Found rosidl_generator_cpp: 3.1.7 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.475s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.481s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.892s] [31mCMake Error at CMakeLists.txt:18 (find_package):
[0.892s]   By not providing "Finddiagnostic_updater.cmake" in CMAKE_MODULE_PATH this
[0.892s]   project has asked CMake to find a package configuration file provided by
[0.892s]   "diagnostic_updater", but CMake did not find one.
[0.892s] 
[0.892s]   Could not find a package configuration file provided by
[0.892s]   "diagnostic_updater" with any of the following names:
[0.892s] 
[0.892s]     diagnostic_updaterConfig.cmake
[0.892s]     diagnostic_updater-config.cmake
[0.892s] 
[0.892s]   Add the installation prefix of "diagnostic_updater" to CMAKE_PREFIX_PATH or
[0.892s]   set "diagnostic_updater_DIR" to a directory containing one of the above
[0.892s]   files.  If "diagnostic_updater" provides a separate development package or
[0.892s]   SDK, be sure it has been installed.
[0.892s] 
[0.892s] [0m
[0.892s] -- Configuring incomplete, errors occurred!
[0.892s] See also "/home/jeanho/gps_rtk/build/ublox_gps/CMakeFiles/CMakeOutput.log".
[0.897s] gmake: *** [Makefile:757: cmake_check_build_system] Error 1
[0.899s] Invoked command in '/home/jeanho/gps_rtk/build/ublox_gps' returned '2': AMENT_PREFIX_PATH=/home/jeanho/gps_rtk/install/ublox_msgs:/home/jeanho/gps_rtk/install/ublox_serialization:/home/jeanho/gps_rtk/install/rtcm_msgs:${AMENT_PREFIX_PATH} CMAKE_PREFIX_PATH=/home/jeanho/gps_rtk/install/ublox_msgs:/home/jeanho/gps_rtk/install/ublox_serialization:/home/jeanho/gps_rtk/install/rtcm_msgs:/opt/ros/humble LD_LIBRARY_PATH=/home/jeanho/gps_rtk/install/ublox_msgs/lib:/home/jeanho/gps_rtk/install/rtcm_msgs/lib:/opt/ros/humble/opt/rviz_ogre_vendor/lib:/opt/ros/humble/lib/x86_64-linux-gnu:/opt/ros/humble/lib:/home/jeanho/acados/lib PYTHONPATH=/home/jeanho/gps_rtk/install/ublox_msgs/local/lib/python3.10/dist-packages:/home/jeanho/gps_rtk/install/rtcm_msgs/local/lib/python3.10/dist-packages:${PYTHONPATH} /usr/bin/cmake --build /home/jeanho/gps_rtk/build/ublox_gps -- -j24 -l24
